pvmismatch/contrib/xlsio/example_workflow/example_workflow.py

```diff
@@ -0,0 +1,45 @@
+import pandas as pd
+import numpy as np
+from matplotlib import pyplot as plt
+from pvmismatch.pvmismatch_lib import pvcell, pvconstants, pvmodule, pvstring, pvsystem
+import os
+import sys
+sys.path.append(os.path.abspath(os.path.join(os.getcwd(), os.pardir, 'xlsio')))
```

Comment:
Thanks. Indeed it is now just a temporary solution.  `pvmismatch.contrib.xlsio import *` only works if xlsio is already fully part of the pvmm package and pvmm is installed with xlsio already in it right? I plan to resolve this issue last, so that I can test the scripts easily.

Reply:
>`pvmismatch.contrib.xlsio import *` only works if xlsio is already fully part of the pvmm package and pvmm is installed with xlsio already in it right?

I don't think so, b/c as long as there's a [dunder module, `__init__.py` in contrib](https://github.com/adambgnr/PVMismatch/blob/master/pvmismatch/contrib/__init__.py) and in xlsio, then it's a [Python package](https://docs.python.org/3/tutorial/modules.html#packages), and Python will pick it up automatically.

But now, looking through your branch I see that you have `pvmismatch/contrib/xlsio/xlsio/xlsio.py` and there's no `__init__.py` in either `pvmismatch/contrib/xlsio` or `pvmismatch/contrib/xlsio/xlsio` so this is why your imports fail.

I suggest you remove the extra xlsio folder, and add `pvmismatch/contrib/xlsio/__init__.py` so that xlsio becomes a sub-package of contrib, and you can import xlsio directly as `from pvmismatch.contrib import xlsio` and then use `xlsio.xlsio.system_layout_to_xls`.

Another suggestion would be to move all of the contents of your `xlsio.py` file to `pvmismatch/contrib/xlsio/__init__.py` and then you can just call `xlsio.system_layout_to_xls`.

Although some consider this to be too implicit, and [Zen of Python](https://www.python.org/dev/peps/pep-0020/) says explicit is better, so as an alternate you can add an API inside of your `pvmismatch/contrib/xlsio/__init__.py`:

```python
# API for xlsio
from pvmismatch.contrib.xlsio import *  # assumes you remove the extra xlsio/ folder from this tree
```
and then you can just call `xlsio.system_layout_to_xls` same as if you moved your code to `__init__.py`.

One more note, you can not use _relative_ imports like the next line below, you have to use absolute or full imports that have the full namespace. You can [use dots as shortcuts for the path](https://docs.python.org/3/tutorial/modules.html#intra-package-references) if that helps, so instead of `from pvmismatch.contrib.xlsio import *` you could use `from ..xlsioimport *` where 2 dots stand in for pvmismatch and contrib.

Reply:
Thanks, now that I use pipenv, the imports you propose work in my workflow as well :D